pkg/asset/agent/installconfig.go

```diff
@@ -309,6 +309,11 @@ func warnUnusedConfig(installConfig *types.InstallConfig) {
 		logrus.Warnf(fmt.Sprintf("%s is ignored", fieldPath))
 	}
 
+	if installConfig.FeatureSet != configv1.Default {
+		fieldPath := field.NewPath("FeatureSet")
+		logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, installConfig.FeatureSet))
```

Comment:
Warning is not enough here IMHO. I think we should error out.

Reply:
Changed to be an error.